.ci_support/linux_64_.yaml

```diff
@@ -17,11 +17,11 @@ fftw:
 gdal:
 - '3.7'
 geos:
-- 3.12.0
+- 3.12.1
 glib:
 - '2'
 hdf5:
-- 1.14.2
+- 1.14.3
```

Comment:
Hmm, conda-smithy picked up the geos 3.12.1 migration, but missed the gdal 3.8 migration. Not sure why.